File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.firefox.webdriver import WebDriver
from selenium.webdriver.support import expected_conditions
from webdriver_manager.firefox import GeckoDriverManager

# setup Firefoxdriver with WebDriverManager
service = FirefoxService(GeckoDriverManager().install())

# start the drive
with webdriver.Firefox(service=service) as driver:
    # Access the URL
    driver.get("http://v1.training-support.net/")
    # print title of the page
    print("Title of the page is: ", driver.title)

    # locate the webelement
    driver.find_element(By.ID, "about-link").click()
    # print the title of new page
    print("Title of the new page is: ", driver.title)
    # close the browser
    driver.close()

File: Selenium/Activities/Activity1.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        //setting up the driver
        WebDriverManager.firefoxdriver().setup();
        //Initializing Webdriver
        WebDriver driver = new FirefoxDriver();
        //Access the URL
        driver.get("http://v1.training-support.net/");
        //Get the title of the page and print in
        System.out.println("Title of the page is: "+driver.getTitle());
        //locate the Webelement
        WebElement element = driver.findElement(By.id("about-link"));
        //click on button
        element.click();
        //get the title of the new page and print
        System.out.println("Title of the new page is: "+driver.getTitle());
        //close the browser
        driver.close();

    }
}
